types/twitch-ext/index.d.ts

```diff
@@ -448,6 +448,11 @@ declare namespace Twitch.ext {
          */
         token: string;
 
+        /**
+         * JWT that can be used for front end API requests.
```

Comment:
Let's add a link that is used in the documentation?

```suggestion
         * JWT that can be used for front end API requests.
         * @see {@link https://dev.twitch.tv/docs/extensions/frontend-api-usage Using the Twitch API in an Extension Front End}
```

<img width="517" alt="Снимок экрана 2021-09-02 в 18 03 38" src="https://user-images.githubusercontent.com/10235949/131868204-2ba0116d-aded-44ea-8e45-ff9fd428bb46.png">

Reply:
Hah, seems I rushed the merge a little - your message was added literally 2 minutes after merge. Sorry for that. For me it wasn't so obvious we want to put a link there - seemed to be an unnecessary detail that you'll need to maintain when documentation updates, and URL is lo longer correct.

Cheers!